src/resolvers/resolver.ts

```diff
@@ -15,6 +16,34 @@ export const resolvers = {
       }
       return user;
     },
+    async users(_: unknown, args: { input: Pagination }, context): Promise<UsersListOutput> {
+      getUserId(context.token);
+
+      const skip = args.input?.skip ?? 0;
+      const limit = args.input?.limit ?? 5;
+
+      if (skip <= 0) {
+        throw new CustomError('Skip must be greater than 0', 400);
+      }
+
+      if (limit < 0) {
+        throw new CustomError('Limit must be a positive number', 400);
+      }
+
+      const [users, totalUsers] = await User.findAndCount({ skip, take: limit, order: { name: 'ASC' } });
+      const after: number = totalUsers - skip - limit;
```

Comment:
```suggestion
      const after = totalUsers - skip - limit;
```

Nesse tipo de expressão mais trivial o Typescript já infere o tipo da variável.

Reply:
Perfeito, corrigido